### PR TITLE
Task 5 get api/articles

### PR DESCRIPTION
**Pull request by UArukonda**

### CODE REVIEW

File: db/app.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+const {
+  getTopics,
+  getArticleById,
+  getArticles,
+} = require("./controllers/controller.js");
 const app = express();
 const endpoints = require("../endpoints.json");
 
-app.use(express.json());
+// app.use(express.json());
```
Comment (nc-haz-mallion) [inline db/app.js:10]: This is not required in your code until there is a POST or PATCH, commented out code should not be included in "production ready" code

File: db/controllers/controller.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  //   let sort_by, order;
+  //   if (req.query) {
+  //     sort_by = req.query.sort_by;
+  //     order = req.query.order;
+  //   }
+  return selectArticles() //sort_by, order
+    .then((articles) => {
+      res.status(200).send({ articles });
+    })
+    .catch((err) => console.log(err));
```
Comment (nc-haz-mallion) [inline db/controllers/controller.js:24]: console logging is not actually handling your error, in order to pass this error into the error-handling-middleware chain you will need to invoke `next(err)`

File: db/models/model.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const db = require("../connection.js");
+
+function selectTopics() {
+  return db.query("SELECT * FROM topics").then((response) => response.rows);
+}
+
+function selectArticles(sort_by = "article_id", order = "ASC") {
+  //   console.log(sort_by);
```
Comment (nc-haz-mallion) [inline db/models/model.js:8]: Please do not commit commented out code.

**Review by nc-haz-mallion — COMMENTED** [review]
There seems to be a lot of commented out code here, I understand not wanting to remove your hard work, but it does reduce readability and it is not always clear why code is commented out. 

Please remove this until you add features like sorting to your API. 

Lovely descriptive commit messages, dare I say too-many commits for this feature. 
Try not to commit each file separately.

**Comment by UArukonda** [comment]
> There seems to be a lot of commented out code here, I understand not wanting to remove your hard work, but it does reduce readability and it is not always clear why code is commented out.
> 
> Please remove this until you add features like sorting to your API.
> 
> Lovely descriptive commit messages, dare I say too-many commits for this feature. Try not to commit each file separately.

I forgot about the commented-out code, I would edit the file from the next commit

**Pull request merged**